koans/ArrayKoans.jl

```diff
@@ -1,2 +1,61 @@
 module ArrayKoans
-end
\ No newline at end of file
+
+function array_1d(n)
+    # return a 1D array of zeros with length n
+end
+
+function array_2d()
+    # return a 2x2 matrix of zeros
+end
+
+function array_shape(array)
+    # return the shape of the array
```

Comment:
Some examples showing what exactly the function should return would help guide the user.

Reply:
Koans should: 
- [ ] Have a description of the problem to solve. In this case, we are trying to write a function which will answer the question "what shape does the given array have?", if possible it would be interesting the koan exercise should include an example of what motivates this problem.
- [ ] the solution should link to the documentation for the function used.